apps/frontend/src/pages/projects/libs/components/project-card/project-card.tsx

```diff
@@ -31,11 +32,25 @@ const ProjectCard = ({
 		onDelete(project);
 	}, [onDelete, project]);
 
+	const lastUpdate = getRelativeDateLabel(project.lastActivityDate);
+
 	return (
 		<div className={styles["project-container"]}>
 			<NavLink className={styles["project"] as string} to={projectRoute}>
 				{project.name}
 			</NavLink>
+			{lastUpdate && (
+				<div className={styles["last-activity-wrapper"]}>
+					<span
+						className={getValidClassNames(
+							styles["last-activity"],
+							styles[lastUpdate.colorClass],
+						)}
+					>
+						{lastUpdate.label}
+					</span>
+				</div>
+			)}
```

Comment:
Maybe I should put this label inside the NavLink (not sure, since NavLink is positioned absolutely), and find another way (besides using `0 33% padding`) to achieve this layout

Reply:
Hm. I think you should redesign markup and styles for this component, so every inner elements (name, activity label, and menu button) are layouted using flex-box.
And in this case instead of `NavLink` we can add `onClick` prop to `project-container` div and use `useNavigate` hook for navigating user to the project.
But this is just my point of view. So let our coaches dicide